include/boost/numeric/ublas/tensor/static_strides.hpp

```diff
@@ -293,4 +294,17 @@ struct basic_static_strides<basic_static_extents<T,Extents...>, Layout>
 
 } // namespace boost::numeric::ublas
 
+namespace boost::numeric::ublas::detail{
+    
+template <class L, class T, T... E>
+struct is_strides< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+
+template <class T, T... E, class L>
+struct is_static< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+
+template <class T, T... E, class L>
+struct is_static_rank< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+
+} // namespace boost::numeric::ublas
```

Comment:
1. I recommend not to put in `detail` namespace.
2. Could we place it in a file called `strides_traits.hpp` with all the other strides traits types ?

Reply:
1. Ok
2. same as above